lib/angular_components.dart

```diff
@@ -219,6 +219,7 @@ export 'theme/module.dart';
 export 'utils/async/async.dart';
 
 /// A convenience list of all Directives exposed by this package.
+@Deprecated('Use the specific directives in your app for smaller code size.')
```

Comment:
Maybe: "Use the specific directives ..." -> "List individual directives used by your app ..."